account_check_deposit/views/account_check_deposit_view.xml

```diff
@@ -86,7 +86,7 @@
                             context="{'currency': currency_id,
                                         'journal_id': journal_id}"
                         >
-                            <tree>
+                            <tree create="false">
```

Comment:
maybe is better to define a default value for `currency_id` in `account.check.deposit` rather than forbid to create from wizard. 

i.e.:
```
def _default_currency_id(self):
     return self.env.user.company_id.currency_id
```

What you think?

Reply:
Hi Giovanni, thanks for your advices. The error is while trying to creating an account.move.line object, if I add a default currency in account.move.line I have other errors.
![image](https://user-images.githubusercontent.com/93990941/237060328-c9270042-4145-47d8-9ad6-62f8376fddf8.png)
Company will be empty so I have the same problem.
![image](https://user-images.githubusercontent.com/93990941/237061297-f1fbd3dd-d014-4baf-8309-e3a48ba27144.png)